config/parser.go

```diff
@@ -23,15 +22,11 @@ func Parse(configPath string) (*Config, error) {
 }
 
 // Load Config data from string input
-func LoadFromString(data string) (Config, error) {
-	var b bytes.Buffer
-	b.WriteString("providers:\n")
-	b.WriteString(data)
```

Comment:
Corner case for the provider config generation that's been moved into that function instead.